CHANGELOG

```diff
@@ -1,3 +1,12 @@
+Version 0.12.0 (2020-12-15)
```

Comment:
Did we agree to include the version in the commit message, like "Prepare for 0.12.0 release"?

Reply:
not sure, let me double check and modify if so